content/docs/develop/installation/setup-keptn/index.md

```diff
@@ -49,11 +45,11 @@ Select one of the following options:
     CLUSTER_NAME=nameofcluster
     ZONE=us-central1-a
     REGION=us-central1
-    GKE_VERSION="1.12.7-gke.10"
+    GKE_VERSION="1.18.8-gke.10"
```

Comment:
There is no version 1.18.8

Reply:
Seems like a typo. Will change it to 1.12.8